Makefile

```diff
@@ -3,6 +3,7 @@ release:
 ifndef VERSION
 	$(error VERSION is undefined)
 endif
+	pip install twine
```

Comment:
let's document this as a requirement (can be a simple comment on the Makefile) so we can avoid triggering the install every time we run this task.